src/map/mob_modifier.h

```diff
@@ -100,6 +100,7 @@ enum MOBMODIFIER : int
     MOBMOD_NO_LINK            = 69, // If set, mob cannot link until unset.
     MOBMOD_NO_REST            = 70, // Mob cannot regain hp (e.g. re-burrowing antlions during ENM).
     MOBMOD_LEADER             = 71, // Used for mobs that follow a defined "leader", such as Ul'xzomit mobs.
+    MOBMOD_ENCROACH_TARGET    = 72, // How close a mob will encroach on it's target, attempting to make model to model contact. Encroach distance * 10
```

Comment:
I don't think "encroach" is a good name for this.

Reply:
https://www.thesaurus.com/browse/encroach

I'm in for MOVE_CLOSE, MOVE_TO_MODEL, etc. more descriptive words vs a more uncommon word that _technically_ describes whats going on

Reply:
`MOBMOD_CLOSE_TO_TARGET`    ?

Reply:
I might err on the side of `MOBMOD_MOVE_CLOSE_TO_TARGET` so it sounds more like an instruction, rather than a question

Reply:
> I might err on the side of `MOBMOD_MOVE_CLOSE_TO_TARGET` so it sounds more like an instruction, rather than a question

kinda long.

but also brings me back to including "distance" in the phrasing.

Reply:
I'd rather have a longer name where its extremely clear what it does, rather than a shorter name where there's any doubt

Reply:
`TARGET_DISTANCE_OFFSET` was suggested on discord chat. I think that's clear.

Reply:
Bumping this. Was there ever a mod name decided upon so that this could be implemented if that was the reason it was not merged?

Reply:
`TARGET_DISTANCE_OFFSET` since nobody objected to that and the other comment needed addressed https://github.com/LandSandBoat/server/pull/2888#discussion_r988431034

The PR appears to have been abandoned so someone will need to adopt it and preserve the originators authorship for the work they already did.